File: Cache/Data Structures/Node.cs
namespace DS
{
    public class Node
    {
        public Node? Prev { get; set; }
        public Node? Next { get; set; }
        public string Value { get; set; }
        public Node(){}
        public Node(string value)
        {
            Value = value;
            Prev = null;
            Next = null;
        }
    }
}

File: Hashmap implementation/src/Node.java
public class Node<K, V> {

    public Node(K key, V value) {
        this.key = key;
        this.value = value;
    }

    public Node getNext() {
        return next;
    }

    private final K key;

    public K getKey() {
        return key;
    }

    private V value;

    public void setValue(V value) {
        this.value = value;
    }

    public V getValue() {
        return value;
    }

    private Node next;

    public void setNext(Node next) {
        this.next = next;
    }
}
